### DISCUSSION TITLE
Importing images with require doesn't work anymore in Next.js 10

### DISCUSSION THREAD

**Discussion by cbusoi** [vercel/next.js#18551] [Help] [answered]
# Bug report

## Breaking change?

After upgrading Next.js 9.6 to 10, importing images with `require` doesn't work anymore. Probably a file loader in webpack was removed. 

## To Reproduce
1. Create a new Next.js app...
2. Add in a page this and put a image with this filename in the right place:
 
`<img src={require('../public/img/somepicture.jpg')} />`

3. Starting Next.js outputs:

```
error - ./public/img/somepicture.jpg 1:0
Module parse failed: Unexpected character '�' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
(Source code omitted for this binary file)
```

## Expected behavior

To render the picture

**Comment by timneutkens** [comment, marked as answer]
Based on the steps to reproduce this sounds correct, we never supported requiring images.

**Reply by cbusoi** [reply]
Curious ... so then another package added the functionality ;) Thank you for the fast reply!

**Reply by gurupal** [reply]
@timneutkens 
Why I am getting this weird error while passing image data. Image "src" is already defined.

Uncaught Error: Image is missing required "src" property. Make sure you pass "src" in props to the next/image component. Received: {"width":300,"height":500}

<div className={styles.image_container}> <Link href={/p/${encodeURIComponent(product.slug)}}> <a> <Image src={product.displayImage} alt={product.name} width={300} height={500} /> </a> </Link> </div>
